pkg/cmd/roachtest/test_runner.go

```diff
@@ -1115,7 +1116,15 @@ func (r *testRunner) runTest(
 
 	s := t.Spec().(*registry.TestSpec)
 
-	grafanaAvailable := roachtestflags.Cloud == spec.GCE
```

Comment:
Nice, was wondering why there still weren't metrics on AWS/azure after your previous change but this looks like this was suppressing the message.